src/bci_build/package/kiwi.py

```diff
@@ -0,0 +1,66 @@
+"""KIWI Appliances Builder SDK container for easy appliance building on SLE Micro."""
+
+from bci_build.package import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import BuildType
+from bci_build.package import DevelopmentContainer
+from bci_build.package import OsContainer
+from bci_build.package import OsVersion
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+from bci_build.package import _build_tag_prefix
+from bci_build.package.versions import get_pkg_version
+from bci_build.package.versions import to_major_minor_version
+from bci_build.package.versions import to_major_version
+
+KIWI_CONTAINERS = [
+    DevelopmentContainer(
+        name="kiwi",
+        package_name="kiwi-image",
+        pretty_name="KIWI Appliance Builder (kiwi)",
+        custom_description="{pretty_name} container {based_on_container}. {podman_only}",
+        from_image=f"{_build_tag_prefix(os_version)}/bci-base:{OsContainer.version_to_container_os_version(os_version)}",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        # kiwi is not L3 supported
+        # support_level=SupportLevel.L3,
+        version=(kiwi_ver := get_pkg_version("python3-kiwi", os_version)),
+        version_in_uid=False,
+        additional_versions=[
+            to_major_minor_version(kiwi_ver),
+            to_major_version(kiwi_ver),
+        ],
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description=f"%%kiwi_{ver}%%",
+                package_name="python3-kiwi",
+                parse_version=ver,
+            )
+            for ver in (ParseVersion.MAJOR, ParseVersion.MINOR)
+        ],
+        license="GPL-3.0-or-later",
+        package_list=[
+            "timezone",
+            "lvm2",
+            "xfsprogs",
+            "e2fsprogs",
+            "btrfsprogs",
+            "python3-kiwi",
+            "python3-devel",
+            "python3-pip",
+            "checkmedia",
+            "jing",
+            "iproute2",
+            "dracut-kiwi-oem-repart",
+            "libxml2-devel",
+            "glibc-devel",
+            "enchant-devel",
+            "gcc",
+            "tack",
+            "make",
+            "kiwi-systemdeps-core",
+            "netcat-openbsd",
+        ],
+        build_recipe_type=BuildType.DOCKER,
+    )
+    for os_version in (OsVersion.SP5, OsVersion.SP6)
```

Comment:
Why not build for tumbleweed? We ship the new and shiny kiwi 10 there :-)

Reply:
because the original requestor only needs the SP5 container. but okay, I can add it.